src/AWS/Types.hs

```diff
@@ -15,6 +15,12 @@ import Data.Aeson.Types (typeMismatch)
 import Data.Text (Text)
 
 
+type Fqdn = Text
```

Comment:
what is `Fqdn`?

Reply:
Fully Qualified Domain Name

Reply:
https://en.wikipedia.org/wiki/Fully_qualified_domain_name